warehouse/query-core/src/main/java/datawave/query/tables/content/ContentQueryTable.java

```diff
@@ -54,7 +54,9 @@ public class ContentQueryTable extends BaseQueryLogic<Entry<Key,Value>> {
     private static final String ALL = "\u10FFFF";
 
     private int queryThreads = 100;
-    private ScannerFactory scannerFactory;
+
+    ScannerFactory scannerFactory;
```

Comment:
Does the unit test require public access? I would prefer this stay private and any tests use the getter

Reply:
fair point